### DISCUSSION TITLE
Ezra Bible App 1.11

### DISCUSSION THREAD

**Discussion by tobias-klein** [ezra-bible-app/ezra-bible-app#914] [General]
**Ezra Bible App 1.11** features support for Bible commentaries as well as various bugfixes and enhancements.

Note-worthy features and improvements:
* Add support for Bible commentary modules. ([#886](i886))
* Search terms are not highlighted if search term is found within text that quotes Jesus. ([#889](i889))
* Merge module description and module details tabs in app info dialog. ([#893](i893))
* Module search is hanging if search starts or ends with whitespace. ([#888](i888))

A detailed listing of all changes can be found in the [changelog](https://github.com/ezra-bible-app/ezra-bible-app/blob/master/CHANGELOG.md).

Installation instructions can be found [here](https://ezrabibleapp.net/installation).

[i886]: https://github.com/ezra-bible-app/ezra-bible-app/issues/886
[i893]: https://github.com/ezra-bible-app/ezra-bible-app/issues/893
[i888]: https://github.com/ezra-bible-app/ezra-bible-app/issues/888
[i889]: https://github.com/ezra-bible-app/ezra-bible-app/issues/889

<hr /><em>This discussion was created from the release <a href='https://github.com/ezra-bible-app/ezra-bible-app/releases/tag/1.11.0'>Ezra Bible App 1.11</a>.</em>

**Comment by BeForgiven-Info** [comment]
Tobias:
Thank you very much for your hard work and basically excellent program. Xiphos has suddenly stopped working due to graphics driver incompatibility, so I am in an emergency situation so to speak. I have most of it figured out, but: Questions:
1. Is there a user manual?
2. How do I increase the font size for commentaries and dictionaries? 
3. (running Debian 11, XFCE 4.16, all updates current.)

Many thanks!

**Reply by tobias-klein** [reply]
Hi @BeForgiven-Info,

1. There is no user manual as of now. Which specific aspects are not self-explanatory and would benefit from specific explanations in a user manual? I was also thinking about creating some YouTube videos that explain the various features. A user manual in web-based format could be added to the website of Ezra Bible App.
2. Increasing the font size is not implemented for the side panels. Would it make sense to synchronize this with the font size setting in the Bible text area?

**Comment by BeForgiven-Info** [comment]
Tobias:

Thank you for your reply.

1. I am just in the habit of always reading the manual first. I was not 
sure what I was missing. This is just personal to me, but being deaf, 
videos do me no good; I need printed material. As it is, I am fairly 
comfortable with the program already. A web manual and link in the 
program would be fine; it would automatically give the user a separate 
window for the manual and not hide some or all of the program.

2. My eyes are aging. What is comfortable in one pane for someone will 
be a good choice overall for a minimum. I would suggest that some may 
want to scroll less in the left pane, favoring a smaller font than the 
Bible window. So one setting for all would be great for me, but perhaps 
separate left and right settings may be better, assuming people spend 
most time in the Bible text.

Thanks again.

Tom

Tom Sullivan
***@***.***
FAX: 815-301-2835

**Reply by tobias-klein** [reply]
@BeForgiven-Info 
Thanks for providing some more context, Tom!

Something that we could do for 2) is
a) to have an option that synchronizes the font size of the Bible text with the font size used in the side panel. That would be fairly easy to implement. Then the font size would always be the same both in the side panel and Bible browser and you would control it centrally.
b) The other possibility would be a separate font size setting for the side panel, but that would also require a separate font size control menu / button.

I would opt for a) as a solution to your problem. Would that work for you?

**Reply by bellgamin** [reply]
Option a) would certainly work for me and be GREATLY appreciated.
Also, it would be most helpful if there were a drop-down menu for commentaries, the same as there is for Bible translations.

BTW -- I was using Windows (e-Sword) for years but wanted to switch to Linux. As a Bible teacher, the only thing Linux lacked (versus Windows) was a great Bible app. Then I found Ezra. Now that Ezra has commentaries, it's just what I needed to finally make the move to Linux. And Ezra's Tabs component is an amazing bonus. Thank you very much-- you are truly gifted!!!

**Comment by BeForgiven-Info** [comment]
Tobias:

Yes, thank you very much.

Tom

Tom Sullivan
***@***.***
FAX: 815-301-2835

**Reply by tobias-klein** [reply]
Ok, I added #921 to further track this request!

**Reply by tobias-klein** [reply]
@BeForgiven-Info 
#921 is available now in the [latest release 1.12](https://github.com/ezra-bible-app/ezra-bible-app/releases/tag/1.12.0)!
